board/batocera/fsoverlay/etc/init.d/S31sixad

```diff
@@ -15,8 +15,8 @@ case "$1" in
         fi
         ;;
     stop)
-        killall sixad-bin &> /dev/null
-        killall sixad-sixaxis &> /dev/null
+        killall sixad-bin > /dev/null
+        killall sixad-sixaxis > /dev/null
```

Comment:
You need to send stderr to /dev/null too.

Reply:
So `2>/dev/null` should be enough

Reply:
Or `2>&1 >/dev/null`